OCAT/client/resources/js/components/Assessments/new.js

```diff
@@ -1,16 +1,102 @@
 import React from 'react';
+import { useForm } from "react-hook-form";
 import { AssessmentService } from '../shared/services/assessment.service';
 
 export function AssessmentNew(){
+  const createAssessmentObject = (data)=>{
+    var assessment ={
+      instrument: data.instruments, 
+      catName : data.catName,
+      catDateOfBirth : data.catDateOfBirth,
+    }
+    assessment.score = calculateScore(data)
+    assessment.riskLevel = calculateRiskLevel(assessment.score)
+    return assessment;
+  }
 
-    //create a form that utilizes the "onSubmit" function to send data to OCAT/client/libs and onto the OCAT/server/routes express API
-    const onSubmit = async (data) => {
-        await AssessmentService.submit(data);
-    };
+  const calculateScore = (data)=>{
+    var score = 0;
+    score += +data.prevJudSys;
+    score += +data.phsAltCat;
+    score += +data.prsAltOwn;
+    score += +data.plyDog;
+    score += +data.hssStr;
```

Comment:
I would come up with a different naming convention for this.  You want to make it as easy as possible for another developer to read/understand your code.  Since I understand the app, I know you're taking the sum of the scores here, but the "prevJudSys" name doesn't convey that unless I have knowledge of the assessment question.  I'd suggest "response1" or something more generic.

Reply:
I changed this as recommended.